cardano-testnet/src/Testnet/Process/Cli.hs

```diff
@@ -151,17 +117,8 @@ data ByronDelegationCert
 
 type TmpDir = FilePath
 
-newtype File a = File {unFile :: FilePath}
```

Comment:
This name is confusing, as we're using API's `File` more frequently in testnet. I've opted to removing this type and to use API one.